pkg/kubelet/kubelet.go

```diff
@@ -91,6 +95,7 @@ type volumeMap map[string]volume.Interface
 // New creates a new Kubelet for use in main
 func NewMainKubelet(
 	hostname string,
+	containerRuntimeChoice string,
```

Comment:
Just wrap that with an `if`, I think the logic should be put in a separate func after we merge docker and container runtime together.